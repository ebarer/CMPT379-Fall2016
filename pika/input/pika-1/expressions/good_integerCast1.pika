exec {
	const v := 3.1415.
	const vapprox := [v | int].
	print v; vapprox, _n_.
}