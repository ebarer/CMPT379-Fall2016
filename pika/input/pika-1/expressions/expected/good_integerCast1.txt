3.1415 3
